packages/@aws-cdk-testing/framework-integ/test/aws-rds/test/integ.rolling-instance-updates.ts

```diff
@@ -17,10 +17,12 @@ class RollingInstanceUpdateTestStack extends cdk.Stack {
     });
 
     new rds.DatabaseCluster(this, 'DatabaseCluster', {
-      engine: rds.DatabaseClusterEngine.AURORA,
```

Comment:
Only running the integ test before the snapshot update failed, so I deleted the snapshot directory and ran the test.

```
Engine version not found: oscar/null
```